lib/Common/Exceptions/ReportError.cpp

```diff
@@ -5,7 +5,7 @@
 #include "CommonExceptionsPch.h"
 
 #ifdef _MSC_VER
-__inline
+inline
```

Comment:
Just remove the inline altogether here.  VC does not inline function with try/except anyway.